nvim/init.lua

```diff
@@ -69,6 +69,142 @@ require('lazy').setup({
     opts = {}
   },
 
+  {
+    "folke/zen-mode.nvim",
+    dependencies = { "folke/twilight.nvim" },
+    event = "VeryLazy",
+  },
+
+  {
+    "tris203/hawtkeys.nvim",
+    cmd = { "Hawtkeys", "HawtkeysAll", "HawtkeysDupes" },
+    config = true,
+    dependencies = {
+      "nvim-lua/plenary.nvim",
+      "nvim-treesitter/nvim-treesitter",
+    },
+  },
+
+  {
+    "gen740/SmoothCursor.nvim",
```

Comment:
This plugin seems unnecessary, probably I will create an optional plugins file.

Reply:
Thank you for the feedback I think it make sense Can you provide an example how can be done

Reply:
I think I will go with creating a separate folder "optionalplugins" and putting them there. I hesitate a bit with this because I really want to avoid this config becoming bloatware.

Reply:
@imbacraft let me find some time and will create a pr with the suggestion. If you have some spare time please go ahead and do it. The optional folder can be some interface by adding extra programming languages out off scope of dusk but easy do be added like in my case with Python and C++ They  working flawlessly with dusk and I was really amazed how easily integrated it.
I really love how dusk is only a shell on the top of nvim and how is using only out of the box solutions.